source/text/labelgeometry.ts

```diff
@@ -79,21 +115,23 @@ export class LabelGeometry extends Geometry {
 
         /* Please note the implicit bind in attribEnable */
 
-        /* quadVertex */
-        this._buffers[0].attribEnable(indices[0], 2, gl.FLOAT, false, 0, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[0], 0);
-        /* texCoords */
-        this._buffers[1].attribEnable(indices[1], 4, gl.FLOAT, false, 4 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[1], 1);
-        /* origin */
-        this._buffers[2].attribEnable(indices[2], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[2], 1);
-        /* tangent */
-        this._buffers[3].attribEnable(indices[3], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[3], 1);
-        /* up */
-        this._buffers[4].attribEnable(indices[4], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[4], 1);
+        this._vertices.attribEnable(this._vertexLocation,
```

Comment:
Since the parameter `indices` is unused now, can you specify that in the method doc?

Reply:
Same for `unbindBuffers()`